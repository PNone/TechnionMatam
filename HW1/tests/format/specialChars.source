bar$@K Regev 30 20:40
__first name__ 15 20:30
SPECIAL CHAR 100 20:20