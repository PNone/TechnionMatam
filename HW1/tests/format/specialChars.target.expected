BlockChain Info:
1.
Sender Name: bar$@K
Receiver Name: Regev
Transaction Value: 30
Transaction timestamp: 20:40
2.
Sender Name: __first
Receiver Name: name__
Transaction Value: 15
Transaction timestamp: 20:30
3.
Sender Name: SPECIAL
Receiver Name: CHAR
Transaction Value: 100
Transaction timestamp: 20:20
